### ISSUE TITLE
How to label bags of multiple instances?

### ISSUE THREAD

**Issue by Monikshah** [jakubmonhart/mil_pytorch#2]
So a bag consists of several instances lets say 5 instances, each length 512. The bag should have one label. eg:

bag1 = [[1,2,...4,5],[3,4,...,5,3],[7,8,...,7,9],[3,4,...5,9],[4,5,...,3,2]]
this bag has lable 1. How do I use this data in this code ?

Please help me. 
The examples provided labels each instances and puts them in two separate bag but for multi instance learning, all the instances in a bag does not have labels individually rather the bag as a whole has a label so the description is very confusing.

Thank you!

**Comment by jakubmonhart** [comment]
In the example https://github.com/jakubmonhart/mil_pytorch/blob/master/examples/musk.py , data are loaded along with ids and **instance labels**:

```
instance_labels = pandas.read_csv(filepath_or_buffer='data/musk/labels.csv', sep='\t', header=None).values.reshape(-1)
instance_labels = torch.tensor(instance_labels)
```

For this particular dataset, **bag label**  is 1 if any of its instances is labeled 1 and 0 otherwise:
```
# Create bag labels from instance labels
bagids = torch.unique(ids)
labels = torch.stack([max(instance_labels[ids==i]) for i in bagids]).float()
```
Dataset is then created using the **bag labels**:
```
# Create dataset and divide to train, valid and test part
dataset = mil.MilDataset(data, ids, labels, normalize=True)
```
which then returns one label for each bag.
